package.json

```diff
@@ -28,6 +28,6 @@
   "homepage": "https://github.com/homegamesio/homegames#readme",
   "dependencies": {
     "ws": "^5.2.1",
-    "squishjs": "^0.4.2"
+    "squishjs": "0.4.23"
```

Comment:
0.4.23 is the version I built with effects & onclick changes and all that. We should build and publish a new version of squishjs based on your PR (https://github.com/homegamesio/squish/pull/2) and then make another PR for homegames that updates the squishjs dependency to that version and update all of the usage of `Colors.js`